evm_arithmetization/src/cpu/columns/general.rs

```diff
@@ -180,6 +193,18 @@ pub(crate) struct CpuStackView<T: Copy> {
     pub(crate) stack_len_bounds_aux: T,
 }
 
+/// View of the first `CpuGeneralColumn` storing the product of the negated
```

Comment:
Extreme nit: while factually correct, I find it just a tiny bit confusing to say it contains the product of the op flag and `is_not_kernel_mode`. The op flag only indicates whether or not this view is active, which the effective value being only `is_not_kernel_mode`.